questdb-rs/src/ingress/ndarr.rs

```diff
@@ -0,0 +1,133 @@
+pub(crate) const MAX_DIMS: usize = 32;
```

Comment:
I'd move this top-level in `questb::ingress` and make it public, then add a doc to it. It might be useful to callers.